myo_multithreading_examp.py

```diff
@@ -1,5 +1,6 @@
 import multiprocessing
```

Comment:
hey remember-- Access control enforces policy such that users cannot act outside of their intended permissions.  please fix, not secure

📄 [OWASP](https://owasp.org/Top10/A01_2021-Broken_Access_Control/)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not secure

Reply:
hey nice work -- this looks good secure

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Secure